### ISSUE TITLE
JSX transform fails when keystone is a dependency of a global package

### ISSUE THREAD

**Issue by erg0dic** [keystonejs/keystone#1865]
```
{ "keystone": "0.3.15" }
```
I have a project my-project that has keystone as a dependency. my-project is packaged and pushed to a private npm registry. For deployment, I do an npm i my-project against the private npm registry. I have a "bin" entry in package.json (https://docs.npmjs.com/files/package.json#bin) that symlinks the entry point to my-project. 

When running my-project via that symlink, I experience JSX parse errors, the same symptoms as #1411 & #1514.

It is not clear to me whether this is an issue with keystone or something upstream (babel? browserify?). I am more than happy to continue investigating but I need some direction.

**Comment by erg0dic** [comment]
I have published an npm module that demonstrates this issue.

```
npm install -g keystone-issue-1865
node $(npm root -g)/keystone-issue-1865/keystone.js
```

I get the following output:
<img width="1179" alt="screen shot 2015-12-03 at 10 50 06 pm" src="https://cloud.githubusercontent.com/assets/5550520/11581463/3f4b2af0-9a10-11e5-9c2b-565fb053898c.png">

**Comment by ericelliott** [comment]
Thanks for publishing the module. =) Confirmed.

**Comment by erg0dic** [comment]
I believe this is due to browserify ignoring files under node_modules/
https://github.com/babel/babelify#why-arent-files-in-node_modules-being-transformed

If I add babelify to the list of browserify transforms in package.json (as suggested in the above link), I no longer see these errors.

PR shortly

**Comment by JedWatson** [comment]
Closed by #1913
